src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -545,169 +559,187 @@ namespace cryptonote::rpc {
     } request;
   };
 
-  //-----------------------------------------------
+  /// RPC: stop_mining
   /// Stop mining on the daemon.
   ///
   /// Inputs: none
   ///
-  /// Output values available from a restricted/admin RPC endpoint:
```

Comment:
I wasn't a big fan of how this was done previously but do we specify if the endpoint is public or restricted somewhere now that this is removed?

Reply:
Yes; the converting script looks for "PUBLIC" in the first line after the comment ends, and if not found, marks it as admin-required.